### PR TITLE
Temporary disable ESIMD/aot_mixed.cpp untill get fixed 'Unsupported required sub group size' issue

### PR DESCRIPTION
**Pull request by dbudanov-cmplr**

### CODE REVIEW

**Comment by pvchupin** [comment]
@dbudanov-cmplr, it seems XFAIL doesn't work as test passes in some configurations.
```
[2022-09-13T19:31:26.768Z] Unexpectedly Passed Tests (1):
[2022-09-13T19:31:26.768Z]   SYCL :: ESIMD/aot_mixed.cpp
```
Can you rework as UNSUPPORTED?
Also please add a comment to the test itself that this is temporary until issue is fixed.

File: SYCL/ESIMD/aot_mixed.cpp [hunk 1]
```diff
@@ -8,6 +8,7 @@
 // REQUIRES: gpu
 // UNSUPPORTED: cuda || hip
 // UNSUPPORTED: esimd_emulator
+// UNSUPPORTED: gpu-intel-gen9
```
Comment (aelovikov-intel) [inline SYCL/ESIMD/aot_mixed.cpp]: I don't think it will fix it on Windows where we switched to TGL recently.

Comment (pvchupin) [inline SYCL/ESIMD/aot_mixed.cpp]:
```suggestion
// Temporary disable everywhere until "Unsupported required sub group size" is fixed in some configurations.
// UNSUPPORTED: windows
// UNSUPPORTED: linux
```
Should we try that as unconditional disable?

Comment (v-klochkov) [reply SYCL/ESIMD/aot_mixed.cpp]: Unconditional disable would be the simplest way for now.
"UNSUPPORTED: gpu-intel-gen9" prevents it from running on Gen9, but I guess the error happening on newer Gen, right?

**Review by pvchupin — DISMISSED** [review]

**Review by pvchupin — APPROVED** [review]

**Review by v-klochkov — APPROVED** [review]

**Comment by dbudanov-cmplr** [comment]
@v-klochkov, @pvchupin could you merge the request. The fails in CI are not related to the modified test.

**Pull request merged**